suites/upgrade/client-upgrade/infernalis-client-x/basic/0-cluster/start.yaml

```diff
@@ -0,0 +1,14 @@
+roles:
+- - mon.a
+  - mds.a
+  - osd.0
+  - osd.1
+- - mon.b
```

Comment:
@dillaman you think something like that?

roles:
- - mon.a
  - mds.a
  - osd.0
  - osd.1
- - client.0